scripts/link-validator.conf

```diff
@@ -24,7 +24,44 @@ site-link-validator {
     "https://mvnrepository.com/artifact/"
     # we haven't yet set up the gradle plugin on plugins.gradle.org
     "https://plugins.gradle.org/plugin/org.apache.pekko.grpc.gradle"
+    # Github links generated by sbt-license-report
+    "http://pholser.github.com/jopt-simple"
+    "https://github.com/googleapis/java-iam/proto-google-common-protos"
+    "https://github.com/netty/netty-tcnative/netty-tcnative-boringssl-static"
+    # Other links generated by sbt-license-report
+    "http://lmax-exchange.github.com/disruptor"
+    # Occasionally returns a 500 Internal Server Error
+    "http://code.google.com/"
   ]
 
-  non-https-whitelist = []
+  non-https-whitelist = [
+    # license report
+    "http://aopalliance.sourceforge.net",
+    "http://asm.ow2.io/",
+    "http://code.google.com/p/",
+    "http://commons.apache.org/proper/",
+    "http://findbugs.sourceforge.net/",
+    "http://fusesource.github.io/jansi",
+    "http://hc.apache.org/",
+    "http://hdrhistogram.github.io/HdrHistogram/",
+    "http://jcp.org/en/jsr/detail?id=250",
+    "http://junit.org",
+    "http://lmax-exchange.github.com/disruptor",
+    "http://openjdk.java.net/legal/gplv2+ce.html",
+    "http://opensource.org/licenses/",
+    "http://www.opensource.org/licenses/",
+    "http://parboiled.org",
+    "http://pholser.github.com/jopt-simple",
+    "http://source.android.com/",
+    "http://www.apache.org/licenses",
+    "http://www.eclipse.org/legal/",
+    "http://www.gnu.org/licenses/",
+    "http://www.jcraft.com/jsch/",
```

Comment:
Move to front,  get the list sorted.

Reply:
This list is temporary, will be removed when more functionality will get added to a forked link-validator